html/js/recorder.js

```diff
@@ -130,10 +130,14 @@
       Recorder.resize(240, 160);
       // need to wait until app is resized before displaying permissions screen
       var permissionCommand = function() {
-        if (options && options.permanent) {
-          Recorder.recorder.permitPermanently();
-        } else {
-          Recorder.recorder.permit();
+        try{
```

Comment:
That's a detail but, in our code style, there should be single space before the `{`. The same below with `catch`.